### PR TITLE
Allow multiplayer.

### PR DESCRIPTION
**Pull request by alpinus4**
Ticker is disabled.
Additionally there has to be added global action queue with addObject action handling.
To be added later. Global Action Queue is needed, and it is being added by @Krystian030  along with tree handling.

### CODE REVIEW

**Comment by alpinus4** [comment]
GlobalActionQueue was added by @Krystian0130 
I fixed add agent button, now adding agent is entirely handled by deltas.

**Review by filip-szweda — CHANGES_REQUESTED** [review]

File: scene/world_updater/AgentCreator.gd [hunk 1]
```diff
@@ -17,10 +17,8 @@ func post_agent(new_agent):
...
 	var new_agent = Agent.instantiate()
 	new_agent.init(_position, _id, _owner, _code)
-	objects_container.add_child(new_agent) # should be handled by deltas but is not TODO (also we dont check if there's something on that position, deltas would solve that)
 	post_agent(new_agent)
-	new_agent._post_populate()
-#	new_agent.queue_free()
+	new_agent.queue_free()
 
 func _on_ui_create_agent_requested(position, code):
 	create_agent(code, position)
```
[the remark this answers is not in the document]
Comment (alpinus4) [reply scene/world_updater/AgentCreator.gd:24]: Currently it is exactly like the second option

```gdscript
var new_agent = Agent.instantiate()
new_agent.init(_position, _id, _owner, _code)
post_agent(new_agent)
new_agent.queue_free()
```

File: entity/action/push/Push.gd [hunk 2]
```diff
@@ -27,7 +27,13 @@ func _execute():
 		object.get_node("AnimatedSprite2D").set_animation(animation)
 	
 	for pushed_object_id in _pushed_object_ids:
-		var pushed_object = Globals.WORLD_UPDATER.objects_container.get_node(str(pushed_object_id))
+		# sometimes we might need to wait a few frames for wood to appear
```
Comment (filip-szweda) [inline entity/action/push/Push.gd:30]: A workaround, but it will do for now :)

File: entity/action/push/Push.gd [hunk 3]
```diff
@@ -27,7 +27,13 @@ func _execute():
...
+		var pushed_object = null
+		var retry_counter = 0
+		while pushed_object == null || retry_counter <= 5:
+			pushed_object = Globals.WORLD_UPDATER.objects_container.get_node(str(pushed_object_id))
```
Comment (filip-szweda) [inline entity/action/push/Push.gd:34]: `get_object_by_id`

**Review by filip-szweda — CHANGES_REQUESTED** [review]

**Review by Krystian030 — APPROVED** [review]

**Review by filip-szweda — APPROVED** [review]
Good job :)

**Pull request merged**